--- plugins/gamipress/libraries/cmb2/includes/rest-api/CMB2_REST_Controller_Boxes.php
+++ plugins/gamipress/libraries/cmb2/includes/rest-api/CMB2_REST_Controller_Boxes.php
@@ -105,8 +105,6 @@
			$boxes_data['css_dependencies'] = array_values( array_diff( $wp_styles->queue, $before_css ) );
		}

-		// TODO: look into 'embed' parameter.
-		// http://demo.wp-api.org/wp-json/wp/v2/posts?_embed
		unset( $boxes_data['fields'] );
		// Handle callable properties.
		unset( $boxes_data['show_on_cb'] );
